stack/lamp/manifest.yaml

```
@@ -4,10 +4,10 @@ slugs:
   - lemp
 name: LAMP
 type: stack
-description: LAMP is an acronym for the operating system, Linux; the web server, Apache; the database server, MySQL; and the programming language, PHP. All four of these technologies are open source, which means they are community maintained and freely available for anyone to use. Developers use LAMP stacks to create, host, and maintain web content. It is a popular solution that powers many of the websites you commonly use today.
+description: LAMP is an acronym for the operating system, Linux; the web server, Apache; the database server, MariaDB; and the programming language, PHP. All four of these technologies are open source, which means they are community maintained and freely available for anyone to use. Developers use LAMP stacks to create, host, and maintain web content. It is a popular solution that powers many of the websites you commonly use today.
 services:
-  - php
-  - mysql
+  - php:8.3
```

Hey @FelipoAntonoff 

Take a look at the explanation I gave you here: https://github.com/goinfinite/os-marketplace/pull/9#discussion_r2008360626

```suggestion
  - php-webserver
```